### ISSUE TITLE
Install failed on laravel 11

### ISSUE THREAD

**Issue by qhoirulanwar** [laravel-shift/blueprint#679]
- Laravel Version: 11
- PHP Version: 8.3.1
- Blueprint Version: 2.9
- Platform: Mac

### Issue:

![screenshot-1710331050381](https://github.com/laravel-shift/blueprint/assets/22901097/26f9a51b-ea5b-48b6-be47-d2a73ddcec52)

**Comment by jarwonozt** [comment]
I also experienced something similar when I wanted to upgrade Laravel 11

**Comment by manojo123** [comment]
Not compatible yet, lets wait for next commit. So far my workaround was to fork the project and change the dependencies on composer.json

**Comment by jasonmccreary** [comment]
Blueprint should be compatible with Laravel 11 from a dependency perspective. However, it will not generate code for Laravel 11 (i.e. Models with the `casts` method).

tl;dr; Try `dev-master`.

**Comment by jasonmccreary** [comment]
Just tagged `v2.10`. This should include Laravel 11 computability.

A reminder, it will not generate models with the new `casts` method. They will still use the `$casts` property, which is compatible with both Laravel 10 and Laravel 11. I will generate the new method when support for Laravel 10 is dropped later this month.
